src/main/resources/application.properties

```diff
@@ -106,7 +108,12 @@ sparkYarnSink.additionalConfs.spark.yarn.principal=
 sparkYarnSink.additionalConfs.spark.shuffle.service.enabled=true
 sparkYarnSink.additionalConfs.spark.dynamicAllocation.enabled=true
 sparkYarnSink.executablesFolder=/
-sparkYarnSink.userUsedToKillJob=
+
+spark.emr.clusterId=
+spark.emr.filesToDeploy=
+spark.emr.additionalConfs=
+spark.emr.awsProfile=
+spark.emr.region=
```

Comment:
So our EMR clusters will need to be shut down each ... let's say month as they are not built for long term use. Do you use any properties hot-swapping or would this need to be "redeployed" every time we have to redeploy EMR?

Reply:
Currently, there is no such mechanism. One way would be to move the clusterId to the workflow config, i.e. the config value would end up in the database and can be updated without an application restart. Otherwise, we'd have to look into what spring offers in terms of properties hot-swapping, esp. if that would be relevant for other applications as well, especially on cloud